src/main/java/com/microsoft/graph/Requests/BaseGraphRequestAdapter.java

```diff
@@ -58,6 +59,7 @@ public enum Clouds {
      * @param graphClientOption the GraphClientOption for use in requests.
      * @param baseUrl the base URL for use in requests.
      */
+    @Suppress(names = "LambdaLast")
```

Comment:
Would this work instead for consistency?

```suggestion
    @SuppressWarning("LambdaLast")
```

Reply:
@Suppress is Kotlin specific I believe. 
I think the other would be for any spotbugs warnings we would encounter, but @Suppress requires a Kotlin specific dependency which suggested to me that it would solve the issues we were encountering.

Reply:
Currently on my phone so I can't search through the code. If my memory serves me well we have already suppressed lambda last in the past via suppress warning. Could you do a quick search through the code to look for a precedent please before we settle this?